Loop/Managers/Alerts/StoredAlert.swift

```diff
@@ -50,10 +50,19 @@ extension StoredAlert {
     public var identifier: Alert.Identifier {
         return Alert.Identifier(managerIdentifier: managerIdentifier, alertIdentifier: alertIdentifier)
     }
-    
+
+    var hasUpdatedModificationCounter: Bool { changedValues().keys.contains("modificationCounter") }
+
+    func updateModificationCounter() { setPrimitiveValue(managedObjectContext!.modificationCounter!, forKey: "modificationCounter") }
```

Comment:
[nit] Similar to the other PR: is there a downside to just checking to make sure `managedObjectContext?.modificationCounter` isn't nil?